### ISSUE TITLE
2 keys: 1 can be updated, the other not

### ISSUE THREAD

**Issue by pjgueno** [solokeys/solo2-cli#102]
I have an issue I can't understand:
I was given 2 solokeys and I was told I have to update them before use.
I followed the instructions and it works perfectly on the first key.

For the second key, I can see 1 blue bink only and then:
![Screen Shot 2023-03-08 at 20 39 08](https://user-images.githubusercontent.com/16778931/223828124-5850c8ee-f789-4464-b0c1-7031908250b2.png)

20:20200101.0 is under 2:20220822.0, isn't it ?

```
MacBook-3:SoloUpdates2 PJ$ solo2 update --verbose
Downloading latest release from https://github.com/solokeys/solo2/
Fetched firmware version 2:20220822.0 (2.964.0)
 INFO  solo2::transport > using CTAP as minimal transport
 INFO  solo2::transport > using CTAP as minimal transport
 INFO  solo2::device    > device fw version: 20:20200101.0
 INFO  solo2::device    > new fw version: 2:20220822.0
Firmware version on device higher than firmware version used.
This would be rejected by the device.
Error: Firmware rollback attempt
```
Can you explain me the issue.

When I plug the both keys:

```
MacBook-3:SoloUpdates2 PJ$ solo2 ls
Solo 2 1949AECA1525A256A384748B4189736D (CTAP only, firmware 2:20220822.0, locked)
Solo 2 FEB84FD1CFABCF59AD7F5C2BDBD8FEA9 (CTAP only, firmware 20:20200101.0)
MacBook-3:SoloUpdates2 PJ$ 
```
The first line is the key which was updated succesfully. Why is there this locked ?

**Comment by JohnLGalt** [comment]
Look at the first number.

**20**:20200101.0
**2**:20220822.0

20 > 2

**Comment by pjgueno** [comment]
But the key were given to me at the same time and the current firmware is 2... How can the 2nd key be higher than the current firmware ? The key doesn t blink or lights except once blue. Can I override ?

**Comment by pjgueno** [comment]
It means solo20 ! Instead of solo2...

**Comment by JohnLGalt** [comment]
I wonder if that is either a custom firmware, or else one of the keys you have is a 'hacker' edition.

**Comment by pjgueno** [comment]
It should be the hacker edition. It was from the kickstarter. Lukas from Sensor.Community who is always up to date (or state of the art?) give us 2 keys each. Unfortunately I have jsut one now...

**Comment by JohnLGalt** [comment]
Yeah, the hacker edition cannot be updated using the normal updates from my understanding.

I have one of each as well. I use the standard one all the time. The hacker edition is sitting here because I haven't learned it / figured it out yet.

**Comment by pjgueno** [comment]
The both keys are the same. The Update perfectly worked for the first. Maybe I will succeed to get a new key ...

**Comment by pjgueno** [comment]
Does anyone have the same issue?

**Comment by JohnLGalt** [comment]
Hacker edition is not the same as regular edition.

Look at your screenshots again - the Solo2 also shows "locked" after it. The Solo20 does not.

I suspect that while they *look* the same they actually are not.

**Comment by pjgueno** [comment]
![image](https://github.com/solokeys/solo2-cli/assets/16778931/4d61d51d-e991-4d3b-af38-2b24a521b50d)

**Comment by pjgueno** [comment]
I can't Tell more...
Both keys came in such carton stuff.
